Cargo.lock

```diff
@@ -17,6 +17,21 @@ dependencies = [
  "memchr",
 ]
 
+[[package]]
+name = "android-tzdata"
```

Comment:
How come we need android-tzdata in the lock file? In general there are many new dependencies that seem not obviously needed. Maybe we can disable some features of the new dependencies?

(I figured it would be better to ask than figuring out for myself, because I never manage to find the time to figure it out by myself.)

Reply:
It's an indirect dependency of `serde_with`'s `std` feature, although curiously it refuses to show up in `cargo tree` regardless of what flags I pass it.

Anyways, it's indeed unnecessary. I've updated the dependency declaration of `serde_with` to exclude it along with a few others.